infrastructure/common.tfvars

```diff
@@ -1,3 +1,7 @@
 project_owner           = "nhs-uec"
 vpc_name                = "sm-vpc"
 vpn_security_group_name = "vpn-sg"
+tag_version             = "1"
+service                 = "uec-ftrs"
+cost_centre             = "P0675"
+
```

Comment:
This is trickier than I thought it would be .... so I am going to propose we get this in and then invite someone like Dave Lavender to review cos we always change it later . That said I think some of the tags set below in both dev.tfvars and test.tfvars could be defined once in common.tfvars. So I think 
- data_type - which after discussing with Tamara should probably be PCD, 
- project_type, 
- public facing and 
- on_off_pattern

data classification can stay in dev and test and remain as 3 (it will be 5 in prod when we get that far)
service_category should apparently be platinum in prod but maybe not that high in dev/test so let's leave it in dev/test tfvars but set it to platinum for now